### ISSUE TITLE
makeStr.py reading lattice point error

### ISSUE THREAD

**Issue by googhgoo** [msg-byu/enumlib#78]
Given a struct_enum.out that goes:

```
    2 # Number of points in the multilattice
  0.0000000       1.4093000      0.81365852        # d01 d-vector, labels: 0
  0.0000000       0.0000000       1.6273200        # d02 d-vector, labels: 1/2/3
```
The first lattice point vector is read as [ 0.0, 1.4093000, 0.81365852, 0.0], because the 0 in
` labels: 0`
is also read as a float as well.
I think changing line 706 from
`                vec = [float(v) for v in temp.split() if RepresentsFloat(v)]`
to
`                vec = [float(v) for v in temp.split()[0:3] if RepresentsFloat(v)]`
fix this issue.

**Comment by glwhart** [comment]
@googhgoo Are you commenting on `pymatgen`? There is no python code associated with `enumlib`. It looks like you are referring to python...

**Comment by googhgoo** [comment]
This file was in aux_src. Maybe it's just a adhoc code for postprocessing.. but I thought I would leave a comment about it.

**Comment by glwhart** [comment]
Oh, I forgot about that. A student wrote that and I haven't ever used it. We'll take a look.

**Comment by wsmorgan** [comment]
This is now fixed in pull request #82.
